src/helpers/getDeckAdvice/advice/HEARTHGUARDS/index.js

```diff
@@ -1,7 +1,12 @@
 const getStructures = cards => cards.filter(c => c.type === 'structure')
+// Construction workers not considered for now; May be added later.
+const structureSpawningCardIds = new Set([
+  "W13",  // Hearthguards
```

Comment:
I think you mean “Rockworkers” here?
```suggestion
  "W13",  // Rockworkers
```